React/accordion-component/src/utils/content.js

```diff
@@ -0,0 +1,24 @@
+ export const accordionData = [
+    {
+      title: 'Section 1',
+      content: `Lorem ipsum dolor, sit amet consectetur adipisicing elit. Quis sapiente
+      laborum cupiditate possimus labore, hic temporibus velit dicta earum
+      suscipit commodi eum enim atque at? Et perspiciatis dolore iure
+      voluptatem.`
+    },
+    {
+      title: 'Section 2',
+      content: `Lorem ipsum, dolor sit amet consectetur adipisicing elit. Mollitia veniam
+      reprehenderit nam assumenda voluptatem ut. Ipsum eius dicta, officiis
+      quaerat iure quos dolorum accusantium ducimus in illum vero commodi
+      pariatur? Impedit autem esse nostrum quasi, fugiat a aut error cumque
+      quidem maiores doloremque est numquam praesentium eos voluptatem amet!
+      Repudiandae, mollitia id reprehenderit a ab odit!`
+    },
+    {
+      title: 'Section 3',
+      content: `Sapiente expedita hic obcaecati, laboriosam similique omnis architecto ducimus magnam accusantium corrupti
+      quam sint dolore pariatur perspiciatis, necessitatibus rem vel dignissimos
+      dolor ut sequi minus iste? Quas?`
+    }
+  ];
```

Comment:
There are two `package-lock.json` files added, one added in React folder itself and it can be removed.
Also if there is one package-lock.json file then yarn.lock is not required, you can remove that as well. If you are worried about those using YARN then you can delete both package-lock and yarn.lock, then both npm and yarn users will be able to generate their own lock files from `package.json`

Reply:
Ok I will do that 
Can you tell how is the UI of this component and also how the formatting will be done?

Reply:
UI is decent👍 component should just be simple and customizable for other people to easily use.

For formatting install prettier extension